scripts/start_online_demo.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+function seedDemoData(){
+    echo 'Seeding data for online demo'
+    bytebase --host http://localhost --port 80 --demo --data /var/opt/bytebase &
+    sleep 10
```

Comment:
I would not chain "sleep 10", instead to provide more context why sleeping 10

echo "Sleep 10 seconds for bytebase to finish migration..."

sleep 10